core/src/main/scala/kafka/common/InterBrokerSendThread.scala

```diff
@@ -34,6 +41,8 @@ abstract class InterBrokerSendThread(name: String,
   extends ShutdownableThread(name, isInterruptible) {
 
   def generateRequests(): Iterable[RequestAndCompletionHandler]
+  var unsentRequests = new UnsentRequests
+  val unsentExpiryMs = AdminClient.DefaultRequestTimeoutMs
```

Comment:
This class is abstract anyway, we could leave this as an abstract `def` (which could still be overridden as a `val`)